x-pack/plugins/alerting/server/rules_client/rules_client.ts

```diff
@@ -878,6 +887,13 @@ export class RulesClient {
         muted: {
           terms: { field: 'alert.attributes.muteAll' },
         },
+        snoozed: {
+          date_range: {
+            field: 'alert.attributes.snoozeEndTime',
+            format: 'strict_date_time',
```

Comment:
Does the format matter here? We're just counting docs, or do you always need to provide one?

Reply:
I got error without it

Reply:
perfect, thanks!